Innovative Flix/force-app/main/default/objects/Season__c/listViews/All.listView-meta.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Add there season number and tv serie title

Reply:
Done